app/monitoring/TipMonitoring.scala

```diff
@@ -0,0 +1,26 @@
+package monitoring
+
+import app.BuildInfo
+import com.gu.tip.{Tip, TipConfig, TipFactory}
+import config.Configuration
+
+object TipMonitoring {
```

Comment:
this is actually a function to turn the configuration into a "Tip" so consider renaming this to `TipFromConfig` and call the `init` function `apply`  Then you can just call it with `TipFromConfig(config)`